source/common/ssl/context_config_impl.cc

```diff
@@ -25,10 +25,13 @@ const std::string ContextConfigImpl::DEFAULT_CIPHER_SUITES =
     "AES256-SHA256:"
     "AES256-SHA";
 
+const std::string ContextConfigImpl::DEFAULT_ECDH_CURVES = "X25519:P-256";
```

Comment:
Yes assuming at least 2 people who know what they are talking about give it a +1 I'm fine changing the defaults.